### PR TITLE
update to psc v0.10.5

### PR DESCRIPTION
**Pull request by soupi**
I have also changed the interaction with pursuit to fit the current api and what i think would be generate useful responses.

also requires your approval for the PR in [purescript-node-irc](/hdgarrood/purescript-node-irc) :)

thanks!

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -12,10 +12,11 @@
     "output"
   ],
   "dependencies": {
-    "purescript-console": "^0.1.0",
-    "purescript-affjax": "^0.5.1",
+    "purescript-console": "^2.0.0",
+    "purescript-affjax": "^3.0.1",
+    "purescript-arrays": "^3.2.0",
+    "purescript-strings": "^2.1.0",
```
Comment (hdgarrood) [inline bower.json]: Could you please use ^1.0.0 of node-irc? I made a release after merging your PR

File: bower.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "holly",
   "description": "An IRC bot for #purescript",
-  "version": "0.1.0",
```
Comment (hdgarrood) [inline bower.json:4]: Could you remove this field please?

**Review by hdgarrood — COMMENTED** [review]
Looks good, just a couple of minor comments :)

**Comment by soupi** [comment]
done.

**Comment by hdgarrood** [comment]
Lovely, thank you!

**Pull request merged**